Source/ThirdParty/CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ if (NOT MINI_URHO)
         endif ()
     endif ()
 
-    if (URHO3D_PROFILING)
+    if (URHO3D_PROFILING AND URHO3D_NETWORK)
```

Comment:
`URHO3D_NETWORK` controls enabling of Urho3D networking features. Tracy does not use any of those. We also may want to profile single player game. We even may want to profile such game over the network. A mobile game would be a prime example. This change is wrong :]

Reply:
Thanks, For some reason I thought that was causing some linker errors I was getting while debugging.  when really the problem must have been from the WIN32 Console Change. Makes perfect sense - reverting..